docs/user/FAQ.rst

```diff
@@ -110,3 +110,44 @@ dictionary. Below, there is example code on how to achieve this:
     # for each sample so that it can be weighted
     net = MyNet(MyModule, ..., criterion__reduce=False)
     net.fit(X, y)
+
+
+I already split my data into training and validation sets, how can I use them?
+------------------------------------------------------------------------------
+
+If you have already defined a training and validation datasets that are
+subclasses of PyTorch :class:`~torch.utils.data.Dataset`, you can use
+:func:`~skorch.helper.predefined_split` to wrap your validation dataset and
+pass it to :class:`~skorch.net.NeuralNet`'s ``train_split`` parameter:
+
+.. code:: python
+
+    from skorch.helpers import predefined_split
+
+    net = NeuralNet(
+        ...,
+        train_split=predefined_split(valid_ds)
+    )
+    # None is passed because train_ds already contains the target
+    net.fit(train_ds, None)
```

Comment:
for `NeuralNet` you don't need to pass `y=None`, this is only necessary for classifiers and regressors.